### ISSUE TITLE
Relations are missing a focus outline on a single item title

### ISSUE THREAD

**Issue by gu-stav** [strapi/strapi#15108]
### Required System information

- Node.js version: not applicable
- NPM version: not applicable
- Strapi version: 4.5.3
- Database: not applicable
- Operating system: MacOS

### Describe the bug

When tabbing through relation fields I don't see a focus outline on the item title, which makes it impossible to understand where my focus is. This is an a11y issue.

### Steps to reproduce the behavior

1. Create a content-type with a relation field
2. Add at least one relation
3. Tab through the relation item(s)

https://user-images.githubusercontent.com/2244375/206164006-63912f1d-9a77-4f11-bf23-49531e4c3282.mp4

### Expected behavior

I expect to see a focus outline similar to repeatable components (video).
